release.sh

```diff
@@ -0,0 +1,144 @@
+#!/bin/bash
+
+# Perform a docker-machine release
+# GITHUB_TOKEN=XXXXX ./release.sh 0.X.X
+
+#GITHUB_USER=docker
+GITHUB_USER=jeanlaurent
+GITHUB_REPO=machine
+
+
+function display {
+    echo "🐳   $1"
+    echo ""
+}
+
+function checkError {
+    if [ "$?" -ne 0 ]; then
+        echo "😡   $1"
+        exit 1
+    fi
+}
+
+VERSION=$1
+GITHUB_VERSION="v$1"
+PROJECT_URL="git@github.com:$GITHUB_USER/$GITHUB_REPO"
+RELEASE_DIR=../release-$VERSION
+DOCKER_MACHINE=$GOPATH/src/github.com/docker/machine/bin/docker-machine
+
+if [ -z "$VERSION" ]; then
+  echo "Missing version argument"
+  exit 1
+fi
+
+if [[ -z ${GITHUB_TOKEN} ]]; then
+    echo "GITHUB_TOKEN missing"
+    exit 1
+fi
+
+if [[ ! -f ${DOCKER_MACHINE} ]]; then
+    echo "You must have a fresh build of docker-machine here : ${DOCKER_MACHINE}"
+    exit 1
+fi
+
+github-release --version > /dev/null 2>&1
+checkError "github-release is not installed, check https://github.com/aktau/github-release, aborting."
+
+display "Starting release $VERSION on $PROJECT_URL with token $GITHUB_TOKEN"
+
+ if [ -d "$RELEASE_DIR" ]; then
+   display "Cleaning up $RELEASE_DIR"
+   rm -rdf $RELEASE_DIR
+   checkError "Can't clean up."
+ fi
+
+ display "Cloning into $RELEASE_DIR from $PROJECT_URL"
+
+ mkdir -p $RELEASE_DIR
+ checkError "Can't create $RELEASE_DIR, aborting."
+ git clone -q $PROJECT_URL $RELEASE_DIR
+ checkError "Can't clone into $RELEASE_DIR, aborting."
```

Comment:
Why not put:

```bash
set -e
```

at the top of the script and remove these error checks?  It will cause `bash` to exit on any failed commands.